meta-mender-qemu/scripts/docker/entrypoint.sh

```diff
@@ -27,11 +27,16 @@ if [ -f /mnt/config/artifact-verify-key.pem ]; then
     CONFIG_ARGS="$CONFIG_ARGS --verify-key=/mnt/config/artifact-verify-key.pem"
 fi
 
+# Extract Docker IP and exclude loopback address.
+DOCKER_IP="$(ip addr | sed -ne '/^ *inet /{/127\.0\.0\.1/d;s/^ *inet  *\([^ ]*\) .*/\1/;p}')"
+
 if [ ! -e /mender-setup-complete ]; then
     ./setup-mender-configuration.py --img="$DISK_IMG" \
                                     --server-url=$SERVER_URL \
-                                    --tenant-token=$TENANT_TOKEN $CONFIG_ARGS
+                                    --tenant-token=$TENANT_TOKEN $CONFIG_ARGS \
+                                    --docker-ip="$DOCKER_IP"
     touch /mender-setup-complete
 fi
 
+export QEMU_NET_HOSTFWD=",hostfwd=tcp::80-:80"
```

Comment:
Don't you also need 021ed34f5d049fcd4f8e4f43b9c620c9a53182e0?

Reply:
Yes, good catch. I reviewed only thud.

Can we wait until thud is updated and then we backport at once? This is anyway for a future feature, and it desn't harm to be incomplete IMO.

Reply:
We can also port the commits from #776 and then re-build and re-test

Reply:
Ah of course. So it's actually my fault! ;-D

Grabbing the commits from #776 sounds good.

Reply:
:+1: